pkg/resourceinterpreter/defaultinterpreter/prune/prune.go

```diff
@@ -62,6 +62,11 @@ func RemoveIrrelevantField(workload *unstructured.Unstructured) error {
 			return err
 		}
 
+		// Never delete an object managed by karmada from member clusters. No need to propagate ttlSecondsAfterFinished.
```

Comment:
> Never delete an object managed by karmada from member clusters.

This description is a bit difficult to understand in the current context. Can you briefly describe why this field was removed?